src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -144,6 +198,33 @@ data class GetParticipantProjectSpeciesResponsePayload(
     val participantProjectSpecies: ParticipantProjectSpeciesPayload,
 ) : SuccessResponsePayload
 
+data class ParticipantProjectForSpeciesPayload(
+    val activeDeliverableId: DeliverableId?,
+    val participantProjectSpeciesId: ParticipantProjectSpeciesId,
+    val participantProjectSpeciesSubmissionStatus: SubmissionStatus,
+    val projectId: ProjectId,
+    val projectName: String,
+    val speciesId: SpeciesId,
+)
+
+data class GetParticipantProjectsForSpeciesResponsePayload(
+    val participantProjectsForSpecies: List<ParticipantProjectForSpeciesPayload>,
+) : SuccessResponsePayload
+
+data class SpeciesForParticipantProjectPayload(
+    val participantProjectSpeciesId: ParticipantProjectSpeciesId,
+    val participantProjectSpeciesRationale: String?,
+    val participantProjectSpeciesSubmissionStatus: SubmissionStatus,
```

Comment:
These names seem pretty long, but I assume this is to clearly distinguish the per-species submission status from the deliverable-level one.

Reply:
Exactly, this model is really a partial combination of two models, specific to a UI view, so I wanted it to be extra clear. Another option is to just pass back both of the entire models in the payload like: 

```
{
  participantProjectSpecies: { ... },
  species: { ... },
}
```